http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriberSpec.scala

```diff
@@ -175,16 +172,12 @@ class LeoPubsubMessageSubscriberSpec
       .inTransaction(followupQuery.getFollowupAction(followupKey))
       .unsafeRunSync() shouldBe None
 
-    val caught = the[WorkbenchException] thrownBy {
-      leoSubscriber.messageResponder(message).unsafeRunSync()
-    }
+    leoSubscriber.messageResponder(message).unsafeRunSync() //no exception is thrown
 
     dbRef
       .inTransaction(followupQuery.getFollowupAction(followupKey))
       .unsafeRunSync() shouldBe None
 
-    caught.getMessage should include("Failed to process StopUpdateMessage")
```

Comment:
I think this exception is still thrown on line 86 of LeoPubsubMessageSubscriber?